testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/web/ListenerTestCase.java

```diff
@@ -199,7 +199,7 @@ public void testHttpConnector() throws Exception {
         // check that the connector is live
         String cURL = "http://" + url.getHost() + ":8181";
         String response = HttpRequest.get(cURL, 10, TimeUnit.SECONDS);
-        assertTrue("Invalid response: " + response, response.indexOf("JBoss") >= 0);
+        assertTrue("Invalid response: " + response, response.indexOf("Welcome") >= 0);
```

Comment:
Since you are touching these lines, this is really from the '90, this should be `response.contains(...)`... just sayin'.

Reply:
2x :)